python/ray/accelerators/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ray.accelerators.accelerators import NVIDIA_TESLA_V100
+
+__all__ = ["NVIDIA_TESLA_V100"]
```

Comment:
See https://github.com/tensorflow/cloud/blob/83f4bdf273bf801bca360ff77b05f63acd5a75cc/src/python/tensorflow_cloud/core/machine_config.py#L28

Reply:
How about we include all the NVIDIA accelerators in that list by default?